config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
 
   root to: 'application#index.html'
-  resources :memberships, only: [:new, :create, :index, :show]
+  resources :memberships, only: [:new, :create, :index, :show, :update]
```

Comment:
Which PR/feature is that? It (the update method in controller and route) is there because I put it there in order for this feature to pass its tests, if it is incorrect the feature was not described well enough.